src/main/java/application/Application.java

```diff
@@ -1,38 +1,102 @@
-        System.out.println("Updating products:");
-        for (Product product : service.getAll()) {
+        productService.update(newCat);
+        for (Product product : productService.getAll()) {
             System.out.println(product);
         }
 
         System.out.println("Deleting products:");
-        service.delete(julia.getId());
-        for (Product product : service.getAll()) {
+        productService.delete(julia.getId());
+        for (Product product : productService.getAll()) {
             System.out.println(product);
         }
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        System.out.println("Creating users:");
+        User firstUser = new User("Tatiana", "tatiana_schwarz", "1111");
+        User secondUser = new User("Zoie", "Zoie_Nikitina", "Stepanych1love");
+        userService.create(firstUser);
+        userService.create(secondUser);
+        for (User user : userService.getAll()) {
+            System.out.println(user);
+        }
+
+        System.out.println("Changing password:");
+        User newUser = new User("Zoie", "Zoie_Nikitina", "StepanychAsshole");
+        newUser.setId(2L);
+        userService.update(newUser);
+        System.out.println(userService.get(2L));
+
+        System.out.println("Deleting user:");
+        userService.delete(2L);
+        try {
+            System.out.println(userService.get(2L));
+        } catch (NoSuchElementException exception) {
+            System.out.println("The user does not exist");
+        }
+
+        ShoppingCartService shoppingCartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        System.out.println("Creating shopping carts:");
+        ShoppingCart firstCart = new ShoppingCart(1);
+        ShoppingCart secondCart = new ShoppingCart(2);
+        shoppingCartService.create(firstCart);
+        shoppingCartService.create(secondCart);
+        System.out.println(firstCart);
+        System.out.println(secondCart);
+
+        System.out.println("Adding products to existing carts:");
+        shoppingCartService.addProduct(firstCart, cat);
+        shoppingCartService.addProduct(secondCart, dog);
+        System.out.println(firstCart);
+        System.out.println(secondCart);
+
+        System.out.println("Clearing cart:");
+        shoppingCartService.clear(firstCart);
+        System.out.println(firstCart);
+
+        System.out.println("Deleting cart:");
+        shoppingCartService.delete(firstCart);
+        try {
+            System.out.println(shoppingCartService.getByUserId(1L));
+        } catch (NoSuchElementException exception) {
+            System.out.println("The cart does not exist");
+        }
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        System.out.println("Completing order:");
+        Order order = orderService.completeOrder(secondCart);
+        System.out.println(order);
```

Comment:
Let's use all of the existing Order's methods

Reply:
done